Sources/NIO/Bootstrap.swift

```diff
@@ -828,6 +935,54 @@ public final class DatagramBootstrap {
             }
         }
     }
+    
+    /// A channel option which can be applied to datagram bootstrap using shorthand notation.
+    /// - See: DatagramBootstrap.channelOptions(_ options: [Option])
+    public struct Option {
+        private let data: ShorthandChannelOption
+        
+        private init(_ data: ShorthandChannelOption) {
+            self.data = data
+        }
+        
+        /// Apply the contained option to the supplied DatagramBootstrap
+        /// - Parameter to: bootstrap to apply this option to.
+        @usableFromInline
+        func applyOption(to bootstrap: DatagramBootstrap) {
+            data.applyOption(to: bootstrap)
+        }
+        
+        fileprivate enum ShorthandChannelOption {
+            case reuseAddr
+            case disableAutoRead
+            
+            func applyOption(to bootstrap: DatagramBootstrap) {
+                switch self {
+                case .reuseAddr:
+                    _ = bootstrap.channelOption(ChannelOptions.socketOption(.reuseaddr), value: 1)
+                case .disableAutoRead:
+                    _ = bootstrap.channelOption(ChannelOptions.autoRead, value: false)
+                
+                }
+            }
+        }
+    }
+}
+
+// Hashable for the convenience of users.
+extension DatagramBootstrap.Option: Hashable {}
```

Comment:
why is it `DatagramBootstrap.Option` for UDP but `NIOTCPShorthandOption` for TCP?

Reply:
what about NIOTS's upcoming UDP support? https://github.com/apple/swift-nio-transport-services/pull/68

Reply:
Renamed to top level NIOUDPShorthandOption